nixos/lib/eval-config.nix

```diff
@@ -36,7 +36,7 @@ let
     _file = ./eval-config.nix;
     key = _file;
     config = {
-      nixpkgs.system = lib.mkDefault system_;
+      nixpkgs.localSystem = lib.mkDefault { inherit system; };
```

Comment:
The ` ? builtins.currentSystem` fallback above is no longer needed. At some future point I'll make this work with an `// optionalAttrs (args ? system) { .. }` instead.